### DISCUSSION TITLE
how i open stage editor / stage viewer.

### DISCUSSION THREAD

**Discussion by MemTheBun** [Leather128/LeathersFunkinEngine#123] [Q&A] [answered]
how i open stage editor / stage viewer.

On build

**Comment by Leather128** [comment, marked as answer]
options > tools > stage edtior

**Reply by MemTheBun** [reply]
but i dont see it

**Reply by Leather128** [reply]
oh, you need the most recent build then (u can get it from actions), aka there isn't a stable release with stage editor yet

**Reply by MemTheBun** [reply]
page pls i dont find it

**Reply by Leather128** [reply]
https://github.com/Leather128/LeatherEngine/actions/runs/1665666096
